iceoryx_posh/test/moduletests/test_roudi_memory_provider.cpp

```diff
@@ -349,7 +349,7 @@ TEST_F(MemoryProvider_Test, SegmentIdValueAfterCreationIsValid)
     ASSERT_THAT(segmentId.has_value(), Eq(true));
     // the segment id being monotonic increasing is an implementation detail, in the case that the implementation
     // changes, just remove this check, since we already check to get a valid result
-    EXPECT_THAT(segmentId.value(), Eq(segmentIdOffset + 1));
+    EXPECT_THAT(segmentId.value(), Eq(segmentIdOffset + 1U));
```

Comment:
Is clang tidy complaining about the missing U? Otherwise we should not introduce them but slowly eliminate them.

Reply:
That will be done on a separate PR. Wanna give it a go? ;)